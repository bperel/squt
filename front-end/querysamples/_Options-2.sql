SELECT ALL SQL_BIG_RESULT SQL_NO_CACHE b.a
FROM b
LOCK IN SHARE MODE;