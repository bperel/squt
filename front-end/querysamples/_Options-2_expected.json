{
	"Tables" : {
		"b" : {
			"b" : {
				"OUTPUT" : {
					"a" : {
						"-1" : "a"
					}
				}
			}
		}
	},
	"Options" : {
		"SQL_NO_CACHE" : 1,
		"TL_READ_WITH_SHARED_LOCKS" : 1,
		"SELECT_BIG_RESULT" : 1
	}
}